engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/ingress/IngressPreparer.scala

```diff
@@ -0,0 +1,60 @@
+package pl.touk.nussknacker.k8s.manager.ingress
+
+import com.typesafe.config.{Config, ConfigFactory, ConfigRenderOptions}
+import monocle.macros.GenLens
+import pl.touk.nussknacker.engine.api.{LiteStreamMetaData, ProcessVersion, RequestResponseMetaData, TypeSpecificData}
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManager.{labelsForScenario, objectNamePrefixedWithNussknackerInstanceNameWithoutSanitization}
+import pl.touk.nussknacker.k8s.manager.K8sUtils.sanitizeObjectName
+import pl.touk.nussknacker.k8s.manager.RequestResponseSlugUtils
+import pl.touk.nussknacker.k8s.manager.ingress.IngressPreparer.rewriteAnnotation
+import play.api.libs.json.Json
+import skuber.networking.v1.Ingress
+import monocle.std.option.some
+
+case class IngressConfig(host: String, config: Config = ConfigFactory.empty())
```

Comment:
Let's add root path with default set to '/' - costs nothing and very usable.

Reply:
I assume you mean root path before slug in ingress address. If so, done.

Reply:
yes, I was talking about root path before slug in ingress address